api/proto/info/info.proto

```diff
@@ -566,6 +624,20 @@ message ZInfoVpn {
   repeated ZInfoVpnConn conn = 10; // Connection Information
 }
 
+enum ZNetworkInstanceRunState {
+    ZNETINST_RUN_STATE_UNSPECIFIED = 0;
+    ZNETINST_RUN_STATE_INIT = 1;
+    // Terminal State.
+    ZNETINST_RUN_STATE_ONLINE = 2;
+    // Terminal State.
+    ZNETINST_RUN_STATE_ERROR = 3;
+    // De-Activation in Progress.
+    ZNETINST_RUN_STATE_HALTING = 4;
+    // Terminal State.
+    // Network Instance De-Activated.
+    ZNETINST_RUN_STATE_HALTED = 5;
```

Comment:
I think we should either have ONLINE/OFFLINE or ACTIVE/INACTIVE. HALTED doesn't make sense in the EVE API for this object.

Reply:
Lets Just remove it? Currently, we don't support this operation for Network Instance - right? We can add proper term when we actually have a need?

Reply:
At the API we can set Activate=false in the config, even if the commercial controller/UI doesn't do that.
Thus we should report a state when status.Activated=false.